docs/ova/README.md

```diff
@@ -0,0 +1,84 @@
+# Cluster API Provider vSphere, OVA Installer Build process
+
+The build process uses a collection of bash scripts to launch a docker container on your local machine
+where we provision a linux OS, install dependencies, and extract the filesystem to make the OVA.
+
+- [Cluster API Provider vSphere, OVA Installer Build process](#cluster-api-provider-vsphere-ova-installer-build-process)
+  - [Usage](#usage)
+    - [Prerequisites](#prerequisites)
+    - [Build bundle and OVA](#build-bundle-and-ova)
+      - [Build script](#build-script)
+  - [Deploy](#deploy)
+    - [Using the bootstrap cluster](#using-the-bootstrap-cluster)
+
+## Usage
+
+The build process is controlled from a central script, `build.sh`. This script
+launches the build docker container and controls our provisioning and ova
+extraction through `make`.
+
+### Prerequisites
+
+The build machine must have `docker`.
+
+- `docker for Mac`: https://www.docker.com/docker-mac
+- `docker for Windows`: https://www.docker.com/docker-windows
+
+### Build bundle and OVA
+
+#### Build script
+
+The build script pulls the desired versions of each included component into the build container.
+
+> You must specify build step `ova-dev` when calling build.sh
+
+If called without any values, `build.sh` will get Kubernetes version: `stable-1`
+```
+./build/build.sh ova-dev
+```
+
+If called with the values below, `build.sh` will include Kubernetes version v1.12.1
+```
+./build/build.sh ova-dev --kubernetes-version v1.12.1
+```
+
+## Deploy
+
+The OVA must be deployed to a vCenter.
+Deploying to ESX host is not supported at the moment, it might work though.
+
+The recommended method for deploying the OVA:
+- Access the vCenter Web UI, click `vCenter Web Client (Flash)`
+- Right click on the desired cluster or resource pool
+- Click `Deploy OVF Template`
+- Select the URL or file and follow the prompts
+- Power on the deployed VM
+- Open the VM Console and wait for the Kubernetes cluster status to be: RUNNING
+
+Alternative, deploying with `govc` (this also requires [`jq`](https://stedolan.github.io/jq/)):
+
+First create a spec to populate the OVA fields during deployment:
+```console
+$ govc import.spec bin/cluster-api-xxxxxx.ova | jq -r . > cluster-api-spec.json
+```
+
+Now edit the file and populate the properties (Root password and SSH key at minimum)
+```console
+$ vi cluster-api-spec.json
+```
+
+Now you can import the OVA on your vSphere infrastructure
+```console
+$ export GOVC_URL='<username>:<password>@<vcenter.fqdn>'
+$ govc import.ova --options=cluster-api-spec.json cluster-api-xxxxxx.ova
+```
+
+And power it on and open the VM Console.
+```console
+$ govc vm.power -on cluster-api
+$ govc vm.console cluster-api
+```
+
+### Using the bootstrap cluster
```

Comment:
Should this be a second level heading? It seems like a step which follows "Deploy", not a sub-step.

```suggestion
## Using the bootstrap cluster
```